ttnn/cpp/ttnn/multi_device.hpp

```diff
@@ -24,4 +22,8 @@ std::vector<ttnn::Tensor> get_device_tensors(const ttnn::Tensor& tensor);
 
 Tensor aggregate_as_tensor(std::vector<Tensor>& tensor_shards);
 
-}  // namespace ttnn::multi_device
+}  // namespace multi_device
+
+using namespace multi_device;
```

Comment:
I saw this pattern multiple times but can't understand it. 
Why have a namespace at all if we expose things like that anyway?

Reply:
The reason why I thought we adopted this pattern in a number of places was to mimic python submoduling.

Is that right @arakhmati @eyonland